drivers/ps2/Kconfig

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 menuconfig PS2
-	bool "PS/2 Drivers"
+	bool "PS/2 drivers"
```

Comment:
Personal System/2?

Reply:
I deliberately left that one out since I doubt many are familiar with the original name for it. It has been just "PS/2" for so long.

I wont object to adding it, though.

Reply:
Fair enough